packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -226,10 +226,10 @@ export const compiledTemplate = (node: t.Expression, sheets: string[], meta: Met
 
   return template(
     `
-  <CC>
-    <CS ${nonceAttribute}>{%%cssNode%%}</CS>
-    {%%jsxNode%%}
-  </CC>
+    <>
```

Comment:
remove fragment + style

Reply:
also remove calls to `hoistSheet` (so there isnt any styles added to the bundle)

Reply:
Yups :)

Reply:
Kept this fragment because it was throwing error for css prop in `compiled.tsx` story

```bash
WARNING in ./examples/compiled.tsx
Module build failed (from ./node_modules/babel-loader/lib/index.js):
ReferenceError: /compiled/examples/compiled.tsx: Container is falsy
```